tests/test_statemanager.py

```diff
@@ -113,6 +117,20 @@ def review_unlock(self):
     def rewind(self):
         self.datetime = datetime.utcnow() - timedelta(hours=2)
 
+    @with_roles(call={'author'})
+    @state.transition(state.UNPUBLISHED, state.PUBLISHED, message=u"Abort this transition")
+    @reviewstate.transition(reviewstate.UNLOCKED, reviewstate.PENDING, title="Publish")
+    def abort(self, success=False):
+        if not success:
+            raise AbortTransition(success, "You shall not pass!")
```

Comment:
Pass only one parameter, a list/tuple if necessary, and expect just that to be returned to caller.